ðŸš€ Inizio processo di pubblicazione...
--------------------------------------
ðŸ”„ Eseguo sync-blog.py...
Traceback (most recent call last):
  File "/Users/teodoro/Documents/ODONTOIATRIA/P-Med Smile/Sito e Blog/sync-blog.py", line 4, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
âž• Aggiungo i file a Git...
